dom/src/main/scala/com/thoughtworks/binding/dom.scala

```diff
@@ -405,6 +414,18 @@ object dom {
           attributes.collectFirst { case (UnprefixedName(`unprefixedName`), Text(text)) => text }
         }
 
+        private def propertyAccess(xmlName: QName, objectAccess: RefTree): Select = {
+          xmlName match {
+            case UnprefixedName(localPart) =>
+              q"$objectAccess.${TermName(NameTransformer.encode(localPart))}"
+            case PrefixedName(prefix, localPart) =>
+              localPart.split(':').foldLeft(q"$objectAccess.${TermName(NameTransformer.encode(prefix))}") {
+                (prefixExpr, segmentName) =>
+                  q"$prefixExpr.${TermName(NameTransformer.encode(segmentName))}"
```

Comment:
This change enables `svg` tags by simply adding an additional `val svg: scalatags.jsdom.SvgTags`.